src/modules/vendor/components/programs/ProfileInfo.vue

```diff
@@ -160,6 +178,32 @@ export default {
       this.$v.newModule.$reset();
       this.moduleCreationModal = false;
     },
+    async openModuleEditionModal (module) {
+      this.editedModule = pick(module, ['_id', 'title']);
+      this.moduleEditionModal = true;
+    },
+    async editModule () {
+      try {
+        this.modalLoading = true;
+        this.$v.editedModule.$touch();
+        if (this.$v.editedModule.$error) return NotifyWarning('Champ(s) invalide(s)');
+
+        await Modules.updateById(this.editedModule._id, pick(this.editedModule, ['title']));
+        this.resetModuleEditionModal();
```

Comment:
en fait ici il faut plutot mettre `this.moduleEditionModal = false;` et avec le `@hide` la methode reset sera appelée automatiquement